cmake/OpenCVCompilerOptimizations.cmake

```diff
@@ -575,9 +568,9 @@ foreach(OPT ${CPU_KNOWN_OPTIMIZATIONS})
     ocv_is_optimization_in_list(__is_from_baseline ${OPT} ${CPU_BASELINE_FINAL})
   endif()
   if(CPU_${OPT}_SUPPORTED)
-    if(";${CPU_DISPATCH};" MATCHES ";${OPT};" AND NOT __is_from_baseline)
+    if(";${CPU_DISPATCH};" MATCHES ";${OPT};" AND NOT __is_from_baseline AND NOT __is_disabled)
       list(APPEND CPU_DISPATCH_FINAL ${OPT})
```

Comment:
`CPU_BASELINE_DISABLE` should not affect `CPU_DISPATCH` modes.

Reply:
Done.